src/modules/test/recipes/CommandRunnerTests.json

```diff
@@ -0,0 +1,34 @@
+[
+    {
+        "ComponentName": "CommandRunner",
+        "ObjectName": "commandStatus",
+        "Desired": false,
```

Comment:
How do we know wether to Set/Get without desired property though? I like this suggestion for the naming though, @MariusNi what do you think?

Reply:
To be explicit it may be easier for others to understand: `"ObjectType": "desired|reported"` instead of assuming a value for desired if it is absent or using desired: true|false. Do you agree @MariusNi, @AhmedBM ?

Reply:
I brought the idea forward with @MariusNi, he likes the `desired: true/false` better. Im agnostic, just want to close this PR :P